docs/guides/admin/docs/modules/plugin-management.md

```diff
@@ -0,0 +1,32 @@
+Plugin Management
+=================
+
+Opencast comes with a number of plugins which allow you to turn additional functionality off and on.
+Plugins can be enabled at runtime in `etc/org.opencastproject.plugin.impl.PluginManagerImpl.cfg` by setting the specific
+configuration to `on` or `off`.
```

Comment:
I would remove the off, since disabling features does currently causes troubles, and you cannot enable a feature by setting the configuration to off.

Reply:
Works for most plugins. The only one it does seem to cause trouble is Theodul. Specifically marked this in the config file.